compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/DerivedConfigUtils.h

```diff
@@ -7,12 +7,18 @@
 #ifndef IREE_COMPILER_CODEGEN_DIALECT_GPU_IR_DERIVEDCONFIGUTILS_H_
 #define IREE_COMPILER_CODEGEN_DIALECT_GPU_IR_DERIVEDCONFIGUTILS_H_
 
+#include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.h"
 #include "mlir/IR/Operation.h"
 
 namespace mlir::iree_compiler::IREE::GPU {
 
 SmallVector<int64_t> deriveThreadTileSizes(Operation *op);
+SmallVector<int64_t> globalLoadDMATileSizes(Operation *op);
 
 } // namespace mlir::iree_compiler::IREE::GPU
 
+namespace mlir::iree_compiler {
```

Comment:
Why not use the `IREE::GPU` namespace?

Reply:
It is actually a declaration of external function. have to use it to avoid Bazel's dep loop complain.